it-tests/BasicFlow.test.ts

```diff
@@ -5,6 +5,8 @@ import { checkEmptyCanvasLoaded, checkTopologyLoaded, openAndSwitchToKaotoFrame,
 import { waitUntil } from 'async-wait-until';
 import * as fs from 'fs-extra';
 
+const DATA_TEST_ID_DATAMAPPERSTEP_2_5 = 'custom-node__route.from.steps.0.step:kaoto-datamapper';
```

Comment:
why marking it with a version 2_5?

Reply:
to know that it was introduced with 2_5, so that we can potentially remove and clean up previous versions more easily when a kaoto version is released